### ISSUE TITLE
Support latex inside html

### ISSUE THREAD

**Issue by EliasSchlatter** [Mathpix/mathpix-markdown-it#79]
Hi Mathpix Team,

With the option htmlTags={true} on a MathpixMarkdown component it is possible to render html, but it seems not to be possible to write latex inside html. Do you think there is a way to implement this capability?

**Comment by OlgaRedozubova** [comment]
> With the option htmlTags={true} on a MathpixMarkdown component it is possible to render html, but it seems not to be possible to write latex inside html. Do you think there is a way to implement this capability?

Yes, you are right, at the moment there is no such possibility, but perhaps it will be implemented in the future.

You can use LaTeX syntax for this:

```
\begin{itemize}
  \item One entry in the list
  \item 
  \begin{align*}
	&\frac{x+4}{(x+1)(x-2)}
  \end{align*}

\end{itemize}

\begin{enumerate}
  \item The labels consists of sequential numbers.
  \item \begin{align*}
	&\frac{x+4}{(x+1)(x-2)}
  \end{align*}
\end{enumerate}
```

You can try it [View Demo](https://htmlpreview.github.io/?https://github.com/Mathpix/mathpix-markdown-it/blob/master/examples/html/input-mmd-to-html.html)

<img width="804" alt="Снимок экрана 2021-01-15 в 17 12 20" src="https://user-images.githubusercontent.com/32493105/104744305-639cf700-5755-11eb-80d8-f59ff293475d.png">


More information about `LaTeX List Syntax`: https://mathpix.com/docs/mathpix-markdown/lists

**Comment by GuVictory** [comment]
@OlgaRedozubova, Please let me know if this functionality is still planned to be implemented in the future

**Comment by OlgaRedozubova** [comment]
LaTeX inside raw HTML blocks is not supported by design — markdown-it treats raw HTML as opaque html_block tokens that bypass inline parsing. Implementing this would require significant post-processing against markdown-it's architecture.

  As a workaround, use LaTeX syntax for lists and structure (`\begin{itemize}`, `\begin{enumerate}`, `\begin{tabular}`) — these fully support math. See https://mathpix.com/docs/mathpix-markdown/lists.
